### DISCUSSION TITLE
blogs/view/syntax-highlight-code-in-NextJS-tailwindcss-daisyui

### DISCUSSION THREAD

**Discussion by giscus** [biplobsd/biplobsd.github.io#7] [Announcements]
# blogs/view/syntax-highlight-code-in-NextJS-tailwindcss-daisyui

Code syntax highlighting is needed to make code more readable and understandable. In this article, y... | Blogs | Biplob Sutradhar

https://biplobsd.me/blogs/view/syntax-highlight-code-in-NextJS-tailwindcss-daisyui.md

**Comment by arnob016** [comment]
Very cool project! 🔥🔥
Keep it up.

**Reply by biplobsd** [reply]
Thanks
